### PR TITLE
KEYCLOAK 12209 Adding note about Z Series

### PR DESCRIPTION
**Pull request by andymunro**
@pskopek  Can you check this?

### CODE REVIEW

File: release_notes/topics/product/7_4_final.adoc [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Red Hat is proud to announce the release of version 7.4 of {project_name} (RH-SSO). RH-SSO is based on the Keycloak project, and enables you to secure your web applications by providing Web SSO capabilities based on popular standards such as OpenID Connect, OAuth 2.0, and SAML 2.0. The RH-SSO server acts as an OpenID Connect or SAML-based identity provider (IdP), allowing your enterprise user directory or third-party IdP to secure your applications via standards-based security tokens.
 
+[NOTE]
+{project_name} for 390x platform (IBM zSeries) is supported only in the OpenShift environment. Bare metal installations on IBM zSeries are not supported.
```
Comment (hmlnarik) [inline release_notes/topics/product/7_4_final.adoc]: Is the S omitted intentionally?

```suggestion
{project_name} for s390x platform (IBM zSeries) is supported only in the OpenShift environment. Bare metal installations on IBM zSeries are not supported.
```

Comment (andymunro) [reply release_notes/topics/product/7_4_final.adoc]: Fixed now. Thanks, Hynek.

**Review by pskopek — APPROVED** [review]

**Comment by andymunro** [comment]
@pskopek Can this be merged now?

**Comment by andymunro** [comment]
@hmlnarik Can you merge this PR?  We need to publish for IBM today and this is part of what we are providing.

**Comment by abstractj** [comment]
@hmlnarik shall we merge this?

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**